cellxgene_schema_cli/cellxgene_schema/write_labels.py

```diff
@@ -209,6 +209,27 @@ def _get_mapping_dict_feature_biotype(self, ids: List[str]) -> Dict[str, str]:
 
         return mapping_dict
 
+    def _get_mapping_dict_feature_length(self, ids: List[str]) -> Dict[str, int]:
+        """
+        Creates a mapping dictionary of feature IDs and feature length, fetching from pre-calculated gene info CSVs
+        derived from GENCODE mappings for supported organisms. Set to 0 for non-gene features.
+
+        :param list[str] ids: feature IDs use for mapping
+
+        :return a mapping dictionary: {id: <int>, id: 0, ...}
+        :rtype dict
+        """
+        mapping_dict = {}
+
+        for i in ids:
+            if i.startswith("ENS"):
```

Comment:
Should we start putting hardcoded string into a constant.py file? Not sure we have enough cases for it to make sense.

Reply:
Might be a good thing to do at the end of the epic? Or we can start now

Reply:
I don't think it's necessary at this point, we can revisit if we find that it's coming up frequently and causing confusion.